guide/how-it-works/custom-spending-conditions.md

```diff
@@ -0,0 +1,225 @@
+   alt-text = "An illustration showing the unlocking scheme of the spending conditions."
+   width = 1600
+   height = 800
+%}
+
+</div>
+
+
+One important thing to note is that, once unlocked, spending conditions always remain active even if additional ones get unlocked. In the above scenario, you can sign a transaction with two keys (condition 1) but also with any one key after six months (condition 2).
+
+## How it works
+
+### Wallet creation and setup
+
+Similar to regular multi-key wallets, a software wallet application or coordination software initiates a multi-key wallet, choosing the number of total keys, and the number of keys required to sign transactions (called threshold). You then add public keys from other wallets generated elsewhere to the multisig after which the software wallet can complete the creation process. Please refer to the [Multi-key](/guide/how-it-works/private-key-management/multi-key/) page for a more detailed overview about how multi-key wallets work.
+
+Notice that all conditions need to be configured during wallet creation. Therefore, you will need to have all public keys ready upfront. If you need to make any changes at a later point in time, you will need to create a new wallet with the desired configuration and move the funds to that wallet.
+
+After the wallet has been created, it needs to be registered on all of the involved signing devices. To do this, you will need the wallet descriptor, which is basically the “map” of the wallet. Each signing device needs to import this map to be able to sign transactions. This process is specific to each signing device and works in the same way as with regular multi-key wallets.
+
+### Timelocks
+
+Timelocks can be relative (e.g. “1 year after the wallet has been last used”) or absolute (e.g. “on January 3rd, 2024”). 
+
+##### Relative timelocks
+
+Relative timelocks can reach up to a maximum of 65535 blocks into the future, which is about 455 days. 
+
+An important aspect of timelocks is that they are applied to each [unspent transaction output (UTXO)](/guide/glossary/#unspent-transaction-output-utxo) in the wallet individually, not to the wallet as a whole. If a wallet uses relative timelocks, this means that the timelocks expire at different times for each UTXO, based on the time and date on which they were deposited into the wallet.
+
+<div class="center" markdown="1">
+
+{% include picture.html
+   image = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-status.png"
+   retina = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-status@2x.png"
+   mobile = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-status-mobile.png"
+   mobileRetina = "/assets/images/guide/how-it-works/custom-spending-conditions/unlock-status-mobile@2x.png"
+   alt-text = "An illustration showing the status of which spending conditions are available for each UTXO."
+   caption = "Figure 2"
```

Comment:
There's mention of a Figure 2 on the page, but not of a Figure 1. Might be easier to just state "in the image". People will know it's the one right next to the text.